precompiles/erc20/integration_test.go

```diff
@@ -588,6 +613,7 @@ var _ = Describe("ERC20 Extension -", func() {
 				// NOTE: we are not passing the contract call here because this test case only covers direct calls
 
 				// FIXME: we have a different error here than the EVM extension
+				// -- says "ERC20: transfer amount exceeds allowance" instead of "authorization not found"
 				Entry(" - through erc20 contract", erc20Call),
```

Comment:
Another case where the ERC20 error messages are not verbose at all - instead of differentiating between the cases where there is actually a spendable balance, that's too small, and then no authorization at all, all these cases throw the same error on ERC20s.. I think if we fully align with ERC20 (without wrapping more info) we create bad UX imho